utils/options.py

```diff
@@ -26,6 +26,11 @@ def parse_options():
                     type=float,
                     help="timeout for test runs in seconds, "\
                          "default: 1.5 * golden runtime")
+    ap.add_argument("--timeout-cc",
+                    dest="timeout_cc",
+                    type=float,
+                    help="timeout for test runs of the cross check in seconds, "\
+                         "default: 1.5 * golden runtime")
```

Comment:
This option seems to be unused?

Reply:
Indeed. I already use it (in check within checker.py) for the cc command. I'll add the usage with one of the next PRs, if that is okay.